samples/net/rpl_border_router/README.rst

```diff
@@ -0,0 +1,136 @@
+.. _rpl-border-router-sample:
+
+RPL Border Router
+#################
+
+Overview
+********
+
+The RPL border router sample application for Zephyr provides a HTTP(S) server
+and net shell for management purposes. Typically it would be used to connect to
+IEEE 802.15.4 network but Bluetooth IPSP network functionality is also possible.
+
+The source code for this sample application can be found at:
+:file:`samples/net/rpl_border_router`.
+
+Requirements
+************
+
+- Real device like Freedom Board (FRDM-K64F) with MCR20A IEEE 802.15.4 support.
+- Linux machine with web browser and the screen terminal emulator (optional).
+- Ethernet access for management purposes (optional).
+
+Note that there is no support for running RPL border router in QEMU, as the
+border router requires access to a real radio network technology like
+IEEE 802.15.4 which is not available in QEMU.
+
+Building and Running
+********************
+
+By default, the integrated HTTP server is configured to listen at port 80.
+If you want to modify the HTTP server config options, please check
+the configuration settings in :file:`samples/net/rpl_border_router/src/main.c`
+file and also in the :file:`samples/net/rpl_border_router/src/config.h` file.
+
+This sample code supports both static and dynamic (DHCPv4) IP addresses that
+can be defined in the project configuration file
+:file:`samples/net/rpl_border_router/prj.conf` :
+
+.. code-block:: console
+
+	CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+	CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
+
+Note that the IPv4 address is only used for connection to the integrated web
+server that provides admin web page for management purposes. The web server
+can also be connected using IPv6 address. If you do not have a management
+interface, then IPv4 support can be disabled in configuration file by setting
+:option:`CONFIG_NET_IPV4` to "n". The RPL router uses only IPv6 when routing
+the network traffic.
+
+.. code-block:: console
+
+	$ make pristine
+	$ make BOARD=<your-board>
+
+By default, the RPL border router application enables net shell support and
+provides some useful commands for debugging and viewing the network status.
+
+.. code-block:: console
+
+	shell> br repair
+	[rpl-br/shell] [INF] br_repair: Starting global repair...
+
+The **br repair** command will cause the RPL network to re-configure itself.
```

Comment:
It's better to put the description of the command before the example rather than after on all of these examples.  For example, 

     The **br repair** command will cause the RPL network to re-configure itself.

     .. code-block:: console

        shell> br repair
        [rpl-br/shell] [INF] br_repair: Starting global repair...